### PR TITLE
Allow to pass custom error object to mockReject and mockRejectOnce

### PR DESCRIPTION
**Pull request by AllenFang**
self explanatory

HI @jefflau please review, I think it's very important that we can pass custom error object to reject mock functions.

Hope we can review and merge it ASAP

Related issue/PR:
https://github.com/jefflau/jest-fetch-mock/pull/30
https://github.com/jefflau/jest-fetch-mock/issues/33

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ describe('Access token action creators', () => {
 
   it('dispatches the correct actions on a failed fetch request', () => {
 
-    fetch.mockReject()
+    fetch.mockReject(new Error('fake error messaeg'))
```
Comment (csi-lk) [inline README.md]: Spelling here `fake error messaeg`

**Comment by hally9k** [comment]
@AllenFang Awesome! 👍

**Comment by AllenFang** [comment]
@csi-lk updated, thanks 👍

**Comment by csi-lk** [comment]
Looks good to me afaict ready to merge @jefflau

**Pull request merged**